src/confd/yang/infix-if-bridge@2024-03-06.yang

```diff
@@ -235,22 +238,28 @@ submodule infix-if-bridge {
     }
   }
 
-  grouping mdb {
-    container mdb {
+  grouping multicast-filters {
+    container multicast-filters {
+      presence multicast;
       description "Bridge multicast database.";
-      config false;
 
-      list entry {
-        description "MDB entry.";
+      list multicast-filter {
+        description "Multicast filter entry.";
         key "group";
 
         leaf group {
-          description "IP multicast group.";
-          type rt-types:ip-multicast-group-address;
+          description "L2/L3 multicast address.";
+          type union {
+            type rt-types:ip-multicast-group-address;
+            type yang:phys-address;
+          }
```

Comment:
Is it possible to use `pattern` with this to verify a `yang:phys-address` starts with 01?

Reply:
I think it better to change to string, and make a pattern. It was my thought, but after this PR.

Reply:
Tried to solve it, succeeded in grep but failed in the pattern. Unclear why? Thats why I posponed it.

Reply:
It's a minor thing, as long as we report any errors in the log if users set unicast mac addresses.

Reply:
Just so we don't constrain this too much: I think we should accept `[0-9a-f][13579bdf]` in the first octet.